src/vl-mapactions-delete-action.js

```diff
@@ -26,7 +26,7 @@ export class VlDeleteAction extends VlBoxSelectAction {
     const style = options ? options.style || defaultStyle : defaultStyle;
 
     const removeFeature = (feature) => {
-      if (feature && layer.getSource().getFeatureById(feature.getId()) === feature) {
+      if (feature && (!feature.getId() || layer.getSource().getFeatureById(feature.getId()) === feature)) {
```

Comment:
Hier zou ik wel zitten `!= null`. Vergeet niet dat in javascript `!0` `true` geeft.

Reply:
Mooi gezien Pieter